app/views/shared/check_answers/_offline_savings_accounts.html.erb

```diff
@@ -1,43 +1,24 @@
 <% read_only = false unless local_assigns[:read_only] %>
-<% if @legal_aid_application.offline_savings? %>
-  <div class="govuk-grid-row" id="app-check-your-answers__offline_savings_accounts">
-    <div class="govuk-grid-column-two-thirds">
-      <h3 class="govuk-heading-m"><%= t(".heading") %></h3>
-    </div>
-    <% if read_only == false %>
-      <div class="govuk-grid-column-one-third align-text-right">
-        <p>
-          <%= govuk_link_to(
-                t("generic.change"),
-                check_answer_url_for(journey_type, :applicant_bank_accounts, @legal_aid_application),
-                class: "change-link",
-                "aria-label": :offline_savings_accounts,
-              ) %>
-        </p>
-      </div>
+<%= govuk_summary_card(title: t(".heading"), html_attributes: { id: "app-check-your-answers__offline_savings_accounts" }, heading_level: 3) do |card| %>
```

Comment:
Where has the conditional been moved to?  It's not in this commit :/

Reply:
it has moved to line 17, but yes I was not sure about this change and what impact or though processes went into it 🤷

Reply:
i think it is just a refactor as the conditional was just being used to render the same summary list but with/without the currency value row.